starter-code/index.js

```diff
@@ -1,17 +1,53 @@
 class SortedList {
-  constructor() {}
+  constructor(...items) {
+    this.items = items;
+    this.length = items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a - b});
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < this.length) {
+      return this.items[pos];
+    } else {
+      throw new Error("OutOfBounds");
+    }
+    
+  }
 
-  max() {}
+  max() {
+    if (this.length > 0) {
+      return Math.max(...this.items);
+    } else {
+      throw new Error("EmptySortedList");
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length > 0) {
+      return Math.min(...this.items);
+    } else {
+      throw new Error("EmptySortedList");
+    }
+  }
 
-  sum() {}
+  sum() {
+      return this.items.reduce((acc, val) => {
```

Comment:
if you will not assign the reduce to a variable, you can also only return at the beginning and it should still give you the correct sum